src/backend/joanie/tests/core/test_api_admin_orders.py

```diff
@@ -0,0 +1,249 @@
+"""Test suite for the admin orders API endpoints."""
+from http import HTTPStatus
```

Comment:
Do you think we should use that strategy for future tests ? IMO I like to see the status code

Reply:
It appears as a good practice by some linter (https://docs.astral.sh/ruff/rules/magic-value-comparison/, https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/magic-value-comparison.html)

That's why I suggest this use. Honestly, I like both, for new comers, using the HTTPStatus interface is maybe more friendly for some exotic code (like 422).

Reply:
Yes, seem's legit. I create the issue